negotiation/message/diagram/contract-agreement-message.puml

```diff
@@ -12,16 +12,14 @@ class "dspace:ContractAgreementMessage" {
     dspace:providerPid : String
     dspace:consumerPid : String
     dspace:agreement : odrl:Agreement
+    dspace:signature: Object
     dspace:callbackAddress : xsd:anyURI
 }
 
 class odrl:Agreement {
     @type : "odrl:Agreement"
     @id: String
-    odrl:target : xsd:anyURI
-    dspace:timestamp : xsd:dateTime
-    dspace:consumerId : String
-    dspace:providerId : String
+    ...
```

Comment:
`dspace:timestamp` was introduced with #82

(including @jimmarino in the conversation)